pkg/systemd/logs.scss

```diff
@@ -80,13 +80,32 @@
     padding: 1.5rem;
 }
 
+.content-header-extra {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+}
+
 .content-header-extra.toggle-filters-closed .filter:not(:first-child) {
     display: none;
 }
 
+.filters-toggle {
+    flex-grow: 1;
+}
+
 .filters-toggle button {
     float: right;
-    padding-top: 8px;
+}
+
+.full-row {
+    flex-basis: 100%;
+    height: 0;
+}
+
+.filter.text-search {
+    min-width: 150px;
```

Comment:
Yeah, we shouldn't use px for sizing and spacing when we can avoid it. It's necessary in media queries (as some browsers don't support rem/em), useful for 1px borders, and we sometimes base icon sizes on it, but otherwise, it's best to not use it.

In this case, it would be better to use em instead of rem, as rem is the root em (the base font size) whereas em is based on the the local font size. If someone is trying to say "it's about this long with the text string", you generally want something like em (or ex or ch), which is based on the character size.